kura/org.eclipse.kura.web2/src/main/java/org/eclipse/kura/web/server/GwtComponentServiceImpl.java

```diff
@@ -59,6 +62,8 @@
     private static final String SERVICE_FACTORY_PID = "service.factoryPid";
     private static final String KURA_UI_SERVICE_HIDE = "kura.ui.service.hide";
 
+    private static final int SERVICE_WAIT_TIMEOUT = 60000;
```

Comment:
🤣 This is a suggestion from my end. You are right, every time the method gets executed the string literal instance will be created and destroyed after the completion but in case of class variable, it will remain in the memory. But it is also a fact that we are also using powerful JVM that hardly cares about such memory optimization. I believe it is just a clean coding approach. Anyways, I am approving this and leaving it to @MMaiero what he believes as a programmer.